### PR TITLE
v3 Proof of Concept

### PR DESCRIPTION
**Pull request by jeremy-visionaid**
Draft proof of concept for v3.0 (#194)

### CODE REVIEW

File: OpenMcdf.Ole/OpenMcdf.Ole.csproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net8.0</TargetFrameworks>
+    <LangVersion>12.0</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
```
Comment (Numpsy) [inline OpenMcdf.Ole/OpenMcdf.Ole.csproj:10]: I'll try to make time to do some tests later, but one small thing here - can these nuget references be conditionalized so they're only included for .NET Standard 2?
(refs comments in https://github.com/ironfede/openmcdf/pull/123 about them being built in to .NET)

**Comment by Numpsy** [comment]
Can we get https://github.com/ironfede/openmcdf/pull/204 into this branch?

File: OpenMcdf.Ole/OpenMcdf.Ole.csproj (hunk 1 above)
Comment (jeremy-visionaid) [reply OpenMcdf.Ole/OpenMcdf.Ole.csproj:10]: @Numpsy Good call, I'd missed that a package was available for it for netstandard2.0.

**Comment by jeremy-visionaid** [comment]
@Numpsy Ah yup, I already ported the code for VT_I8, but the branch was stale

**Comment by Numpsy** [comment]
The v3 draft branch doesn't seem to contain #190 ?

**Comment by jeremy-visionaid** [comment]
This has been superseded by merging to master

**Pull request closed without merging**